packages/skott/bin/cli.ts

```diff
@@ -75,6 +75,12 @@ const cli = sade("skott [entrypoint]", true)
     false
   )
 
+  .option(
+    "-n, --trackTypeOnlyDependencies",
```

Comment:
Currently when providing no options, the `trackTypeOnlyDependencies` is not using the default value **true** provided.

It seems that it's due to the `trackBuiltinDependencies` option already using the shortcut **-n**, which makes the default values for `trackTypeOnlyDependencies` **false**, could you please change that?